packages/core/lib/get-permissions.ts

```diff
@@ -0,0 +1,38 @@
+import { Config, DefaultComponentProps, Permissions } from "../types/Config";
+
+export const getPermissions = ({
+  componentType,
+  globalPermissions,
+  config,
+}: {
+  componentType: keyof DefaultComponentProps;
+  globalPermissions: Partial<Permissions>;
+  config: Config;
+}) => {
+  let componentPermissions = getInitialPermissions({
+    componentType,
+    config,
+    globalPermissions,
+  });
+
+  return componentPermissions;
```

Comment:
I guess this is a `let` because it's going to be enhanced to add the resolver?

Reply:
Correct.